### PR TITLE
Add myself (@JonasTroeltsch) to the list of students

### LINKED ISSUE #13: Add myself (@JonasTroeltsch) to the list of students
I would like to add myself to the list of students in the GitHub organization README.

### PR DESCRIPTION
**Pull request by jonas-tr**

### CODE REVIEW

**Comment by JonasTroeltsch** [comment]
#22

**Comment by jonas-tr** [comment]
[#](https://github.com/heig-vd-dai-course/.github/issues/13)

File: profile/README.md [hunk 1]
```diff
@@ -24,4 +24,5 @@ Please add your name in alphabetical order (by last name) in this format:
 First name Last name [@GitHub username](https://github.com/USERNAME)
 -->
 
+Jonas Troeltsch [@JonasTroeltsch](https://github.com/JonasTroeltsch)
 - _No student yet!_
```
Comment (hadrylouis) [inline profile/README.md]: You can delete this line as there is a student now (you!) :)

**Review by hadrylouis — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 1 above)
Comment (jonas-tr) [reply profile/README.md]: True :)

**Review by hadrylouis — APPROVED** [review]

**Comment by ludelafo** [comment]
You have to link the PR manually to your issue using the right panel on GitHub. But no worries for this time, I'll do it you :)

**Pull request merged**